matchers/match_xml_matcher.go

```diff
@@ -70,7 +70,7 @@ func parseXmlContent(content string) (*xmlNode, error) {
 			if err == io.EOF {
 				break
 			}
-			return nil, fmt.Errorf("failed to decode next token: %v", err)
+			return nil, fmt.Errorf("failed to decode next token: %v", err) // untested section
```

Comment:
Can you explain these commentas a little more? Is this saying there is no test coverage, but you don't want to test it because you are just doing error messages?